### PR TITLE
Fix/mac osx cpu

### PR DESCRIPTION
**Pull request by slackday**
I had problems with clean RSK eating 60% CPU and my project eating 100% (and more). 

Seems webpacks watchpack enters polling mode on Mac OSX without fsevents so I added it as suggested here https://github.com/webpack/webpack-dev-middleware/issues/40

It helped and dropped my CPU usage to stable ~10% on my project and <5% on RSK.

Also webpack seems to watch files in node_modules (which has been fixed) so I added `watchOptions` to webpack.config.js as suggested here https://github.com/JeffreyWay/laravel-elixir-webpack-official/issues/22

I added it but webpack is not my strong side so I don't know if it's in the right place or even needed. Please review so this doesn't cause other problems. 

I tried it in RSK and it works for me.

### CODE REVIEW

**Comment by slackday** [comment]
Oh fsevents causes problems if not on OSX. 

Should I add this to Wiki instead?

File: package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "eslint-plugin-react": "^6.3.0",
     "extend": "^3.0.0",
     "file-loader": "^0.9.0",
+    "fsevents": "^1.0.14",
```
Comment (ClemensSahs) [inline package.json]: i think we will use "optionalDependencies" for this

File: tools/webpack.config.js [hunk 2]
```diff
@@ -43,6 +43,11 @@ const config = {
     sourcePrefix: '  ',
   },
 
+  watchOptions: {
+    poll: false,
+    ignored: /node_modules/,
+  },
+
```
Comment (ClemensSahs) [inline tools/webpack.config.js]: work this on other os without conflicts?

**Comment by langpavel** [comment]
I believe, this is no issue any more. Closing, but Thanks!

**Pull request closed without merging**